frontend/Dockerfile

```
@@ -0,0 +1,26 @@
+FROM node:16-bullseye-slim
+
+RUN apt-get update && \
+    apt-get install -y awscli && \
+    apt-get clean
+
+WORKDIR /app
+
+# Install packages
+COPY package.json yarn.lock ./
+RUN yarn install --frozen-lockfile && yarn cache clean
+
+# Copy code. Annoyingly, you can't copy specific directories all at once. Directory args are
+# always treated as "copy the contents of this directory".
+COPY components components/
+COPY content content/
+COPY data data/
+COPY pages pages/
+COPY public public/
+COPY styles styles/
+COPY next.config.js ./
+
+CMD yarn export && \
+    echo "Uploading to S3..." && \
+    aws s3 sync --delete out/ s3://sitkaproduction/ && \
```

```suggestion
    aws s3 sync --delete --only-show-errors out/ s3://sitkaproduction/ && \
```
2/3rds of the log output for a build task is progress messages from the `s3 sync`, which only takes a few seconds, so we're never going to care about that information.  And it would be easier to see relevant information in the logs (like how long the build took) without it.  `--show-only-errors` should still log anything that might be relevant to our purposes.